lib/ParamedicConfig.js

```diff
@@ -37,7 +37,12 @@ ParamedicConfig.parseFromArguments = function (argv) {
         endPort:           argv.endport || argv.port,
         externalServerUrl: argv.externalServerUrl,
         reportSavePath:    !!argv.reportSavePath? argv.reportSavePath: undefined,
-        cleanUpAfterRun:   !!argv.cleanUpAfterRun? true: false
+        cleanUpAfterRun:   !!argv.cleanUpAfterRun? true: false,
+        logPath:           !!argv.logPath? argv.logPath: undefined,
+        logMins:           !!argv.logMins? argv.logMins: undefined,
+        appName:           !!argv.appName? argv.appName: undefined,
+        simulatorsFolder:  !!argv.simulatorsFolder? argv.simulatorsFolder: undefined,
+        tccDbPath:         !!argv.tccDbPath? argv.tccDbPath: undefined
```

Comment:
Please add defaults for these properties, or set them to `null` instead of `undefined`.